presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -611,18 +613,20 @@ public JoinUsingAnalysis getJoinUsing(Join node)
         return joinUsing.get(NodeRef.of(node));
     }
 
-    public void addTableColumnReferences(Multimap<QualifiedObjectName, String> tableColumnMap)
+    public void addTableColumnReferences(Identity identity, Multimap<QualifiedObjectName, String> tableColumnMap)
     {
+        Map<QualifiedObjectName, Set<String>> identityMap = tableColumnReferences.putIfAbsent(identity, new HashMap<>());
```

Comment:
identityMap is misleading. It conveys that the map contains identities. I'd just call it "references".

Reply:
also, in the previous commit, you made `Map<QualifiedObjectName, Set<String>>` a `LinkedHashMap`, now you made this a `HashMap`